app/models/AtsServiceError.scala

```diff
@@ -16,8 +16,11 @@
 
 package models
 
+import uk.gov.hmrc.http.UpstreamErrorResponse
+
 sealed trait ServiceError
```

Comment:
Possibly bin this and just pass around the UpstreamErrorResponse

Reply:
done. Changed to UptreamErrorResponse